src/lib/theme/type.tsx

```diff
@@ -65,7 +65,7 @@ export function Caption(props: TextProps) {
 }
 
 export function Badge(props: TextProps) {
-  return <TextWrapper className="badge" fontSize={8} fontWeight={600} lineHeight="8px" noWrap />
+  return <TextWrapper className="badge" fontSize="8px" fontWeight={600} lineHeight="8px" noWrap {...props} />
```

Comment:
idk why but passing `{8}` as a number rendered as `72px`, so I passed it in as a string to avoid that.

Reply:
i had this same issue and could not debug either - i had a guess it was something to do with the className